src/error.rs

```diff
@@ -91,18 +92,21 @@ impl From<winapi::um::winnt::HRESULT> for GlyphLoadingError {
 }
 
 /// Reasons why a source might fail to look up a font or fonts.
-#[derive(Clone, Copy, PartialEq, Debug)]
+#[derive(Clone, PartialEq, Debug)]
 pub enum SelectionError {
     /// No font matching the given query was found.
     NotFound,
     /// The source was inaccessible because of an I/O or similar error.
     CannotAccessSource,
+    /// Could not load the font.
+    LoadError(Cow<'static, str>),
```

Comment:
I think it would make more sense for `CannotAccessSource` to carry the error string. Perhaps the `Cow<'static, str>` Could be optional.

Reply:
Sounds good. Went with the optional `Cow<'static, str>`. And I found out I had bunch more [resource-only fonts](https://mastodon.gamedev.place/@shanecelis/111969030028528918).